src/openrct2-ui/input/Shortcuts.cpp

```diff
@@ -739,13 +741,12 @@ static void ShortcutToggleTransparentWater()
 
 using namespace OpenRCT2::Ui;
 
-void ShortcutManager::RegisterDefaultShortcuts()
+// clang-format off
+static const std::vector<std::vector<RegisteredShortcut>> shortcutOrder =
 {
-    // clang-format off
-
-    // Interface
-    RegisterShortcut(ShortcutId::InterfaceCloseTop, STR_SHORTCUT_CLOSE_TOP_MOST_WINDOW, "BACKSPACE", []() { window_close_top(); });
-    RegisterShortcut(ShortcutId::InterfaceCloseAll, STR_SHORTCUT_CLOSE_ALL_FLOATING_WINDOWS, "SHIFT+BACKSPACE", []() {
+    {
+    RegisteredShortcut(ShortcutId::InterfaceCloseTop, STR_SHORTCUT_CLOSE_TOP_MOST_WINDOW, "BACKSPACE", []() { window_close_top();}),
```

Comment:
Now this is a vector, you can drop the `RegisteredShortcut` part, and use list-initialization instead. Then it doesn't look like a function call.

```suggestion
    { ShortcutId::InterfaceCloseTop, STR_SHORTCUT_CLOSE_TOP_MOST_WINDOW, "BACKSPACE", []() { window_close_top();} },
```